compiler/repl-service/client/src/DA/Daml/LF/ReplClient.hs

```diff
@@ -25,6 +25,7 @@ module DA.Daml.LF.ReplClient
 import Control.Concurrent.Async
 import Control.Concurrent.Extra
 import Control.Exception
+import qualified Control.Monad (void)
```

Comment:
I'd pull this from `Data.Functor` instead.

Reply:
Done.